### ISSUE TITLE
Update to ubuntu noble base docker image

### ISSUE THREAD

**Issue by picturedots** [conda-forge/miniforge-images#131]
Similar to https://github.com/conda-forge/miniforge-images/issues/17 the base ubuntu focal is now 4 years old and will be reaching the end of standard support in 7 months (April 2025 -- https://wiki.ubuntu.com/Releases ).  It seems like it's time for an upgrade, especially since not all security patches make it to focal.  For example
`FROM ubuntu:focal-20240530` ->  
`FROM ubuntu:noble-20240801`

**Comment by lurecas** [comment]
I want to give this an upvote, because in my particular case, I have had to update several system libraries (gcc, glibc) in focal that are already present in noble.

**Comment by yangjinhao1234** [comment]
agree! This base image is too old , I must update glibc manually, its annoying.

**Comment by hmaarrfk** [comment]
it is 3 lines for miniforge:
```
    wget --no-hsts --quiet https://github.com/conda-forge/miniforge/releases/download/${MINIFORGE_VERSION}/${MINIFORGE_NAME}-${MINIFORGE_VERSION}-Linux-$(uname -m).sh -O /tmp/miniforge.sh && \
    /bin/bash /tmp/miniforge.sh -b -p ${CONDA_DIR} && \
    rm /tmp/miniforge.sh &
```

the rest you likely already have in your docker images. why not just build your own docker image? why do you need this one?

**Comment by yangjinhao1234** [comment]
> it is 3 lines for miniforge:
> 
> ```
>     wget --no-hsts --quiet https://github.com/conda-forge/miniforge/releases/download/${MINIFORGE_VERSION}/${MINIFORGE_NAME}-${MINIFORGE_VERSION}-Linux-$(uname -m).sh -O /tmp/miniforge.sh && \
>     /bin/bash /tmp/miniforge.sh -b -p ${CONDA_DIR} && \
>     rm /tmp/miniforge.sh &
> ```
> 
> the rest you likely already have in your docker images. why not just build your own docker image? why do you need this one?

I have done. I just want to know why you use this old version. It's meaningless :)

**Comment by yulinor** [comment]
gogogo, update to ubuntu noble base docker image 

There are scenarios where apt is used to install some higher version programs such as libreoffice.

**Comment by picturedots** [comment]
This seems to be now resolved with https://github.com/conda-forge/miniforge-images/pull/145 -- note that is you're referencing the `condaforge/mambaforge` image you won't get the update.
